lib/service/kubernetes.go

```diff
@@ -174,7 +174,13 @@ func (process *TeleportProcess) initKubernetesService(log *logrus.Entry, conn *C
 	}
 
 	// Create the kube server to service listener.
-	authorizer, err := auth.NewAuthorizer(teleportClusterName, accessPoint, lockWatcher)
+	authorizer, err := auth.NewAuthorizer(auth.AuthorizerOpts{
+		ClusterName: teleportClusterName,
+		AccessPoint: accessPoint,
+		LockWatcher: lockWatcher,
+		// TODO(codingllama): Enable after testing.
```

Comment:
Is this "testing" occurring after this PR has been merged (is it reliant on something else)?

Reply:
This is after-merge testing, likely what I'll be doing today.